src/Error/FormattedError.php

```diff
@@ -235,7 +235,9 @@ public static function addDebugEntries(array $formattedError, $e, $debug)
 
         $debug = (int) $debug;
 
-        if ($debug & Debug::RETHROW_INTERNAL_EXCEPTIONS) {
+        $isInternal = ! $e instanceof ClientAware || ! $e->isClientSafe();
+
+        if (($debug & Debug::RETHROW_INTERNAL_EXCEPTIONS) && $isInternal) {
```

Comment:
But this changes the behaviour? What is the rational for this?

What if I may want/desire the current behaviour and not differentiating internal exceptions?